gulpfile.js

```diff
@@ -170,11 +170,17 @@ function delRev(dest, manifestName) {
 
 gulp.task('serve', function(cb) {
   let called = false;
+  let execParams = path.normalize('node_modules/.bin/babel-node');
```

Comment:
Hi @ojongerius thanks for the review.

I can sure call it something like `execCmd` for the sake of not getting confused. But in any case, 

```
if (__DEV__) exec += ' --inspect';
```

This is kinda frowned upon for `if` checks in our code style.

Reply:
I noticed that. I really prefer the look of it. 

However, I do appreciate it can lead to unexpected effects. I've just read https://medium.com/@jonathanabrams/single-line-if-statements-2565c62ff492 and I can see how this can go horribly wrong..